includes/api/validation-functions.php

```diff
@@ -298,11 +298,38 @@ function validate_media_field( $value, array $field ): void {
 	}
 
 	if ( is_field_repeatable( $field ) ) {
-		validate_array(
-			$value,
-			// translators: The name of the field and the field type.
-			\sprintf( \__( '%1$s must be an array of %2$s', 'atlas-content-modeler' ), $field['name'], $field['type'] )
+		// translators: The name of the field.
```

Comment:
Should the translation comment name both placeholders?

```suggestion
		// translators: %1$s: Field name, such as “Title”. %2$s: Field type, such as “string”.
```